styles/components/modal/_core.scss

```diff
@@ -0,0 +1,82 @@
+// Modal structure, taken from ember-modal-dialog/ember-modal-structure
+.ember-modal-overlay {
+  height: 100vh;
+  left: 0;
+  position: fixed;
```

Comment:
total ocd nitpick.. would put this line above the positions so they are grouped together.